packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1872,6 +1884,52 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         ovmStateManager = iOVM_StateManager(address(0));
     }
 
+
+    /*********************
+     * Upgrade Functions *
+     *********************/
+
+    /**
+     * Sets the code of an ovm contract.
+     * @param _address Address to update the code of.
+     * @param _code Bytecode to put into the ovm account.
+     */
+    function ovmSETCODE(
+        address _address,
+        bytes memory _code
+    )
+        override
+        external
+        onlyCallableBy(0x420000000000000000000000000000000000000A)
```

Comment:
Resolving from the L2-compiled AM was a pain here since you need to construct an `ovmCALL` to the AM to load storage from the correct account.  I figured since this will be chugsplash-ified,

Reply:
Think you forgot to finish the sentence :D `chugsplash-ified` -> what does this mean?

Reply:
Maybe better to refactor the address to a constant?

Reply:
Edited -- TLDR: it will be moved to storage.

Reply:
Can we use storage in the execution manager? If not, I think we should just make it a constant.

Reply:
Or an `immutable`.